src/plugins/discourse/mirror.test.js

```diff
@@ -110,63 +188,158 @@ class MockFetcher implements Discourse {
   }
 
   _post(id: PostId): Post | null {
-    const postInfo = this._posts.get(id);
-    if (postInfo == null) {
-      return null;
+    return this._posts.get(id) || null;
+  }
+
+  /**
+   * Adds a new Category.
+   * This will create a category definition topic as well.
+   * The definition topic will create an opening post.
+   *
+   * Note: does not have any perception of subforums.
```

Comment:
By subforum do you mean sub-category relationships?

Reply:
I did mean sub-categories yes. For the current fetcher APIs there's no distinction between them. Only the _real_ `categoryDefinitionTopicIds()` implementation needs to know about it and abstracts the differences away for the mirror.

Reply:
Updated the comment here.